packages/promise-helpers/src/index.ts

```diff
@@ -1,7 +1,7 @@
 export type MaybePromise<T> = Promise<T> | T;
 export type MaybePromiseLike<T> = PromiseLike<T> | T;
 
-const FAKE_PROMISE_SYMBOL_NAME = '@whatwg-node/promise-helpers/FakePromise';
+const kFakePromise = Symbol.for('@whatwg-node/promise-helpers/FakePromise');
```

Comment:
I don't know why that would be a global symbol, so this might as well be:
```suggestion
const kFakePromise = Symbol('@whatwg-node/promise-helpers/FakePromise');
```

Reply:
If you use Symbol, it would be unique and it might be an issue with different versions.